src/main/java/org/dataloader/stats/DelegatingStatisticsCollector.java

```diff
@@ -20,33 +26,63 @@ public DelegatingStatisticsCollector(StatisticsCollector delegateCollector) {
     }
 
     @Override
-    public long incrementLoadCount() {
-        delegateCollector.incrementLoadCount();
-        return collector.incrementLoadCount();
+    public <K> long incrementLoadCount(IncrementLoadCountStatisticsContext<K> context) {
+        delegateCollector.incrementLoadCount(context);
+        return collector.incrementLoadCount(context);
     }
 
+    @Deprecated
     @Override
-    public long incrementBatchLoadCountBy(long delta) {
-        delegateCollector.incrementBatchLoadCountBy(delta);
-        return collector.incrementBatchLoadCountBy(delta);
+    public long incrementLoadCount() {
+        return incrementLoadCount(null);
     }
 
     @Override
-    public long incrementCacheHitCount() {
```

Comment:
The diff is a bit clobbered because I decided to order the methods according to the interface.